stdlib/http/src/main/ballerina/src/http/cookie/http_cookie_client.bal

```diff
@@ -188,7 +188,7 @@ public type CookieClient object {
     # Retrieves the `Response` for a previously-submitted request.
```

Comment:
Can you add code snippets to each of the functions?

Reply:
In most cases, the functions themselves are descriptive and straight forward. And in some cases, there is no point in using the functions isolate. Therefore, IMHO, there's no real need for code snippets in the HTTP module. The scenarios are covered in the Module.md and BBEs.